packages/node-integration-tests/suites/express/sentry-trace/baggage-header-assign/test.ts

```diff
@@ -79,8 +79,7 @@ test('Should populate and propagate sentry baggage if sentry-trace header does n
     test_data: {
       host: 'somewhere.not.sentry',
       baggage: expect.stringContaining(
-        'sentry-environment=prod,sentry-release=1.0,sentry-transaction=GET%20%2Ftest%2Fexpress,' +
-          'sentry-public_key=public,sentry-trace_id=',
+        'sentry-environment=prod,sentry-release=1.0,sentry-public_key=public,sentry-trace_id=',
```

Comment:
Tiny tiny nit: If we're commenting stuff out in the other tests rather than modifying said stuff, I'd leave the old value here, commented out (in addition to the new value, of course). 

(Same in the other spots where this applies.)

Reply:
Done in [5249b77](https://github.com/getsentry/sentry-javascript/pull/5363/commits/5249b77921045befaf7aa4c404ae402866b9db60)